src/styles/theme.scss

```
@@ -12,3 +12,13 @@ $defaultPadding: 8px;
 /** Z-indexes **/
 $layersDrawer: 4;
 $baseMap: 1;
+
+/** Breakpoints **/
+$breakpointMobile: 1024px;
+
+/** Mixins **/
+@mixin mobileBreakpoint {
```

```suggestion
@mixin mobileDownBreakpoint {
```
Add an indicator to id what range the breakpoint is

---

I changed it to `mobileUpBreakPoint` since I'm using `min-width`